blocks/trending-news/trending-news.css

```diff
@@ -0,0 +1,183 @@
+.block.trending-news .link-color {
+    font-size: var(--body-font-size-xxl);
+    line-height: 1.5rem;
```

Comment:
can we use global varriable here ?

Reply:
we are using global for lesser then 1 rem.